src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -229,7 +234,12 @@ def upload(
         files = _get_files(file_name, self.host_mount_path)
         if self.client:
             for file in files:
-                if os.path.isfile(file):
+                is_file_exists_on_remote = self.file_exists_on_remote(
```

Comment:
Either leave off the `is_` part or rename it `does_file_exist_on_remote` (which is better grammatically but too long and clumsy) or just `is_file_on_remote`.

Reply:
Right, `is_file_on_remote` is correct and looks good. Done. Thank you.